guide/popular-topics/webhooks.md

```diff
@@ -157,6 +157,21 @@ client.once('ready', async () => {
 client.login(token);
 ```
 
+::: warning
+If your bot isn't the first webhook in the channel, it will give `[WEBHOOK_TOKEN_UNAVAILABLE]: This action requires a webhook token, but none is available.`
+:::
+
+So, to fix this, we will only try to find our bot with the following code:
```

Comment:
```suggestion
If your bot isn't the first webhook in the channel, it will throw `[WEBHOOK_TOKEN_UNAVAILABLE]: This action requires a webhook token, but none is available.`

To fix this, we will only try to find our bot's webhook with the following code:
:::
```

Also, please include the codeblock and "The code above reads..." *inside* the warning.

Reply:
Oh ok i will thanks